test/aqua/test_qsvm.py

```diff
@@ -165,8 +166,8 @@ def test_qsvm_binary_directly_statevector(self, use_circuits):
                 except Exception:  # pylint: disable=broad-except
                     pass
 
-    @data(False, True)
-    def test_qsvm_setup_data(self, use_circuits):
+    @data('feature_map', 'circuit', 'library')
+    def test_qsvm_setup_data(self, data_encoding_type):
```

Comment:
I see you used the name data_encoding here rather than data_preparation

Reply:
I'll change it to `data_preparation` 👍